engine/src/tangl/mechanics/presence/outfit.py

```
@@ -33,7 +33,9 @@ class OutfitManager(SlottedContainer[Wearable]):
             name=f"{region.name.lower()}_{layer.value}",
             selection_criteria={
                 "predicate": (
-                    lambda wearable, r=region, l=layer.value: r in wearable.covers and wearable.layer.value <= l
+                    lambda wearable, r=region, l=layer.value: (
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)